mindmeld/components/classifier.py

```diff
@@ -444,6 +444,7 @@ def dump(self, model_path, incremental_model_path=None):
                 self._model.dump(path)
```

Comment:
Typos in the comment above statement:
whic --> which
compataible --> compatible